plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/convert.kt

```diff
@@ -140,3 +152,31 @@ internal class To0 : AbstractInterpreter<PluginDataFrameSchema>() {
         return convertImpl(receiver.schema, receiver.columns, typeArg0)
     }
 }
+
+internal abstract class AbstractToSpecificType : AbstractInterpreter<PluginDataFrameSchema>() {
+    val Arguments.functionCall: FirFunctionCall by arg(lens = Interpreter.Id)
+    val Arguments.receiver: ConvertApproximation by arg()
+
+    override fun Arguments.interpret(): PluginDataFrameSchema {
+        val converterAnnotation = functionCall.calleeReference.toResolvedFunctionSymbol()?.getAnnotationByClassId(Names.CONVERTER_ANNOTATION, session)
+        val to = converterAnnotation?.getKClassArgument(Name.identifier("klass"), session)
+        val nullable = converterAnnotation?.getBooleanArgument(Name.identifier("nullable"), session)
+        return if (to != null && nullable != null) {
```

Comment:
this branch is only for proper nullability binding, isn't it?

Reply:
if i understand correctly, yes